keyboards/anavi/knobs3/keymaps/default/keymap.c

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+//#include <stdio.h>
+
```

Comment:
```suggestion
```

Reply:
Thank you for the code review and sorry for the late feedback. I have applied the changed.